src/components/player/index.tsx

```diff
@@ -55,12 +56,32 @@ const VideoResourcePlayer: React.FC<VideoResourcePlayerProps> = ({
   children,
   onFullscreenChange,
   onLoadStart,
+  newNotes,
   ...props
 }) => {
   const {setPlayerPrefs, getPlayerPrefs} = useEggheadPlayerPrefs()
 
   const {subtitle, playbackRate, volumeRate} = getPlayerPrefs()
 
+  const noteTrack = React.useRef<HTMLTrackElement>(null)
+
+  React.useEffect(() => {
+    const track = noteTrack.current?.track
+
+    if (track && newNotes) {
+      console.log(newNotes)
+      newNotes.forEach((note) => {
```

Comment:
we load notes via that api when a lesson loads, but these are NEW notes so we don't want to reprocess the entire text track, instead we can just format them as `VTTCue` objects and chuck them into the track.

BAM 💥